modules/apps/29-fee/keeper/events.go

```diff
@@ -37,7 +37,11 @@ func EmitIncentivizedPacketEvent(ctx sdk.Context, packetID channeltypes.PacketId
 			sdk.NewAttribute(types.AttributeKeyAckFee, totalAckFees.String()),
 			sdk.NewAttribute(types.AttributeKeyTimeoutFee, totalTimeoutFees.String()),
 		),
-	)
+		sdk.NewEvent(
```

Comment:
I never know when this should be added and when not. Is there some rationale?

Reply:
yeah its pretty dumb, needs to just get automated at the SDK level: https://github.com/cosmos/cosmos-sdk/issues/11868